game-core/src/main/java/games/strategy/engine/lobby/server/LobbyContext.java

```diff
@@ -1,22 +0,0 @@
-package games.strategy.engine.lobby.server;
-
-import games.strategy.engine.config.lobby.LobbyPropertyReader;
-
-/**
- * Container for object creation, useful for managing shared dependencies.
- * Provides somewhat similar functionality as a dependency injection framework.
- */
-public final class LobbyContext {
```

Comment:
`ClientContext` was mildly successful to help the client code. `LobbyContext` tried to mirror that and did not work. Whacking this class is a good move :+1: